source/_posts/2020-07-01-release-113.markdown

```diff
@@ -0,0 +1,853 @@
+---
+layout: post
+title: "0.112: Beta release notes"
+description: "0.113: Beta release notes"
+date: 2020-07-14 00:00:00
+date_formatted: "July 22, 2020"
+author: Franck Nijhof
+author_twitter: frenck
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-07-0.113/social.png
+---
+
+<a href='/integrations/#version/0.113'><img src='/images/blog/2020-07-0.113/social.png' style='border: 0;box-shadow: none;'></a>
+
+Beta release notes for 0.113.0
+
+Subjects ideas/structure:
+
+- Automations
+  - Modes
+  - Loops
+  - Chooser
+  - Bonus: Cooldown
+- Speed!!
+  - New optimizations
+  - Python 3.8 (specially optimized build)
+- New MQTT Features?
+
+## Lorem ipsum
+
+<p class='img'>
+<img src='/images/blog/2020-07-0.112/datetime-picker.png' alt='Screenshot of lorem ipsum'></a>
+Screenshot of lorem ipsum.
+</p>
+
+## MDI icons updated
+
+It has taken some time for us to upgrade to the newest version of Material Design Icons, 5.3.45, there was a reason for that, version [5.0.45](https://dev.materialdesignicons.com/upgrade#4.9.95-to-5.0.45) contains a lot of breaking changes.
+We wanted to handle these well so it tooks some time.
+
+A lot of icons are renamed, and some were removed.
+In this release we included all new, and all removed icons. Also will the new and the old name work.
```

Comment:
`Also will the new and the old name work.`
I don't understand what this section means